src/App/Platforms/Android/Autofill/CredentialHelpers.cs

```diff
@@ -237,5 +242,53 @@ private static JSONObject MapExtensionsToJson(Fido2CreateCredentialExtensionsRes
 
             return extensionsJson;
         }
+
+        public static async Task<string> LoadFido2PriviligedAllowedListAsync()
+        {
+            try
+            {
+                using var stream = await FileSystem.OpenAppPackageFileAsync("fido2_priviliged_allow_list.json");
+                using var reader = new StreamReader(stream);
+
+                return reader.ReadToEnd();
+            }
+            catch (Exception ex)
+            {
+                return null;
+            }
+        }
+
+        public static async Task<string> ValidateCallingAppInfoAndGetOriginAsync(CallingAppInfo callingAppInfo, string rpId)
+        {
+            if (callingAppInfo.Origin is null)
+            {
+                return await ValidateNativeAppAndGetOriginAsync(callingAppInfo, rpId);
```

Comment:
🤔 Not a blocker: Maybe rename Native to AssetLinks to align with Google?

Reply:
Sure, could I do that on the PR for asset links verification to save merging time?